hoomd/mpcd/CollisionMethod.cc

```diff
@@ -148,7 +141,8 @@ void mpcd::CollisionMethod::drawGridShift(unsigned int timestep)
     else
         {
         // PRNG using seed and timestep as seeds
-        hoomd::RandomGenerator rng(hoomd::RNGIdentifier::CollisionMethod, m_seed, timestep / m_period);
+        hoomd::RandomGenerator rng(hoomd::Seed(hoomd::RNGIdentifier::CollisionMethod, timestep, seed),
+                                   hoomd::Counter());
```

Comment:
With the current implementation of MPCD, there is only allowed to be one collision method in the system, so this does not technically need an instance ID. However, I am worried that I will forget about this discussion if I ever change that :-), so could you please add a setable instance for this class too?